packages/backend/src/helpers/fileNameHash.ts

```diff
@@ -0,0 +1,7 @@
+import crypto from 'crypto'
```

Comment:
the logic like this should be in: attachment.service.ts
There is no need to create separate file for it